app/src/ui/tab-bar.tsx

```diff
@@ -56,9 +56,9 @@ export class TabBar extends React.Component<ITabBarProps, {}> {
     direction: 'next' | 'previous',
     index: number
   ) => {
-    const children = this.props.children as ReadonlyArray<JSX.Element> | null
+    const children = React.Children.toArray(this.props.children)
```

Comment:
nice!

Reply:
For the future reader, the changes in `<TabBar/>`  were needed to fix [the issues](https://github.com/desktop/desktop/pull/9842#issuecomment-632272679) that @tierninho reported.

They were caused by having a child in `<TabBar/>` that is `false`, which is quite common pattern in JSX when wanting to conditionally render stuff:

```jsx
<TabBar>
  <span>Item 1</span>
  {showSecondItem && (<span>Item 2</span>)}
</TabBar>
```

`React.Children.toArray()` filters out nullable children, plus it converts the structure to an actual array (`this.props.children` is actually an array-like object, and does not implement all `Array.prototype` methods).